Tribler/Test/Community/Multichain/statistics/test_page_rank.py

```diff
@@ -1,3 +1,6 @@
+"""
+Tests for the incremental PageRank.
+"""
 from unittest import TestCase
 
 from networkx import gnm_random_graph
```

Comment:
My bad, I thought it was only being used for the GUI visualization before switching over to Javascript